<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CR5CurrencyCrossRateHistoryFilter</class>
 <widget class="QWidget" name="CR5CurrencyCrossRateHistoryFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date range</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="C5DateEdit" name="leD1">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="C5DateEdit" name="leD2">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Currency 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="leCurrency">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="C5LineEditWithSelector" name="leCurrencyName">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Currency 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leCurrency_2">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="C5LineEditWithSelector" name="leCurrencyName_2">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
